src/Components/Web/src/PublicAPI.Unshipped.txt

```diff
@@ -121,6 +121,7 @@ static Microsoft.AspNetCore.Components.Web.RenderMode.WebAssembly.get -> Microso
 virtual Microsoft.AspNetCore.Components.HtmlRendering.Infrastructure.StaticHtmlRenderer.RenderChildComponent(System.IO.TextWriter! output, ref Microsoft.AspNetCore.Components.RenderTree.RenderTreeFrame componentFrame) -> void
 virtual Microsoft.AspNetCore.Components.HtmlRendering.Infrastructure.StaticHtmlRenderer.WriteComponentHtml(int componentId, System.IO.TextWriter! output) -> void
 virtual Microsoft.AspNetCore.Components.RenderTree.WebRenderer.GetWebRendererId() -> int
+virtual Microsoft.AspNetCore.Components.RenderTree.WebRenderer.UpdateApplicationState(string! applicationState) -> System.Threading.Tasks.Task!
```

Comment:
Why is this needed at all? Previous versions didn't require it, and I don't see why this would be something that is required now.

Reply:
With the new blazor web we prerender every page. This allows to update the application state. We discussed it with Mackinnon and Steve on the email thread

Reply:
Can't find the email thread, can you point me to it?

Reply:
forwarded the email to you